Services/CO.CDP.OrganisationInformation.Persistence/OrganisationInformationContext.cs

```diff
@@ -23,16 +23,51 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 a.Property(ai => ai.Scheme);
                 a.Property(ai => ai.LegalName);
                 a.Property(ai => ai.Uri);
+                a.Property(ai => ai.CreatedOn).IsRequired().HasDefaultValueSql("CURRENT_TIMESTAMP");
+                a.Property(ai => ai.UpdatedOn).IsRequired().HasDefaultValueSql("CURRENT_TIMESTAMP");
```

Comment:
@glgoaco do we need to handle each of these timestamps manually? 

I thought this part takes care of it based on the IDate interface: 

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/a9743f4545e74c78d307b6d505679a4de005f179/Services/CO.CDP.OrganisationInformation.Persistence/OrganisationInformationContext.cs#L78-L87

Reply:
for Owned type object it need to be configured inside as Organisation has already implemented IEntityDate interface.

Reply:
Ok, the idea behind these timestamps was to implement it in a way we don't have to add too much code manually. It seems to have gone the other direction. It looks like it's a problem for owned entities.

Let's not try to solve it in this PR. I'll make a note to reconsider the implementation separately.